### PR TITLE
Various spelling tweaks and small changes in beta release notes

### PR DESCRIPTION
**Pull request by hmmbob**
## Proposed change

Various spelling tweaks and small changes in beta release notes (targeting `rc` branch)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2021-05-05-release-20215.markdown [hunk 1]
```diff
@@ -503,7 +506,7 @@ The following attributes are no longer available:
 
 {% details "Shelly" %}
 
-The Shelly Wi-fi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
+The Shelly Wifi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
```
Comment (frenck) [inline source/_posts/2021-05-05-release-20215.markdown]: This is actually incorrect.

```suggestion
The Shelly Wi-Fi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
```

See:
- <https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>
- <https://developers.google.com/style/word-list#letter-w>

File: source/_posts/2021-05-05-release-20215.markdown [hunk 2]
```diff
@@ -666,7 +669,7 @@ which now using the host udev over all containers.
 For now, we have a fallback ENV variable `USING_UDEV=1` which you can set on your container
 to get the old behavier and give you time for using the docker way to map/use devices.
 
-([@pvizeli]- [])
+([@pvizeli]- [#180])
```
Comment (frenck) [inline source/_posts/2021-05-05-release-20215.markdown:672]:
```suggestion
([@pvizeli]- [#180])

[#180]: https://github.com/home-assistant/docker/pull/180
```

File: source/_posts/2021-05-05-release-20215.markdown [hunk 3]
```diff
@@ -1178,6 +1181,7 @@ their data, which is not allowed for core integrations.
 
 {% enddetails %}
 
+[#180]: https://github.com/home-assistant/docker/pull/180
```
Comment (frenck) [inline source/_posts/2021-05-05-release-20215.markdown]: This list is generated :)
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_posts/2021-05-05-release-20215.markdown (hunk 3 above)
Comment (hmmbob) [reply source/_posts/2021-05-05-release-20215.markdown]: Cool, didn't know that!

**Review by frenck — APPROVED** [review]
Thanks, @hmmbob 👍 


../Frenck 🚂

**Pull request merged**